src/main/java/com/minecolonies/coremod/commands/ActionMenuState.java

```diff
@@ -116,6 +116,21 @@ public EntityPlayerMP getPlayerForArgument(@NotNull final String argumentName)
         return null;
     }
 
+    public EntityPlayerMP getOnlinePlayerForArgument(@NotNull final String argumentName)
+    {
+        for (final ActionArgument actionArgument : getAllArgumentsList())
+        {
+            if (argumentName.equals(actionArgument.getName()))
+            {
+                if (ActionArgumentType.ONLINE_PLAYER == actionArgument.getType())
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1066)